src/client/components/Label/Tags/label.style.css

```diff
@@ -0,0 +1,16 @@
+.button {
```

Comment:
Can you use the class name `.label` instead of a button? This isn't clickable at all.